addons/editor-dark-mode/stage.css

```diff
@@ -1,18 +1,3 @@
-[class*="monitor_monitor-container_"],
-[class*="monitor_list-body_"] {
-  background-color: var(--editorDarkMode-page);
-  color: var(--editorDarkMode-page-text);
-}
-[class*="monitor_list-index_"] {
-  color: var(--editorDarkMode-page-text);
-}
-[class*="monitor_label_"],
-[class*="monitor_list-empty_"] {
-  color: var(--editorDarkMode-page-text) !important;
-}
-
-[class*="monitor_list-header_"],
-[class*="monitor_list-footer_"],
```

Comment:
i dont think this should be removed. dark mode should still apply changes unless this addon is enabled, where in that case, this addon should override darkmode.

Reply:
I guess that makes sense since the setting is still there for other things.